### DISCUSSION TITLE
Documentation for nodes in flowise

### DISCUSSION THREAD

**Discussion by lancerkind** [FlowiseAI/Flowise#1819] [Q&A]
Flowise nodes are drag and drop. Great! Videos about how to create chatbots! Awesome! But how does one find documentation on the various configurations/parameters of these nodes? For example, the InMemory Vector Store has two kinds of output: Memory Retriever and Memory Vector Store. (Image attached.) What do those two options mean? Which should I use? How am I to break to the next level beyond what quick start YouTube videos cover?

(idea: if a link to the node documentation came up when I click on the node's "i" icon, that would be handy!)

Here is the code but as documentation but it doesn't clarify matters either:
https://github.com/FlowiseAI/Flowise/blob/89307b13c9c888b166989eb2379be0f2ced5bab7/packages/components/nodes/vectorstores/InMemory/InMemoryVectorStore.ts#L8
<img width="282" alt="Screenshot 2024-02-26 at 17 36 16" src="https://github.com/FlowiseAI/Flowise/assets/4351057/989014ba-173b-40ff-b2fe-446bc600d4b6">

**Comment by mmiscool** [comment]
I am also looking to better understand object types each node has for input and output.

**Comment by taowang1993** [comment]
yes, we need better docs for sure

**Comment by Nordm80** [comment]
I have the same question. Anyone figured it out yet?

**Comment by bkoprivica** [comment]
This is an abandoned project
